### PR TITLE
get latest

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**

### CODE REVIEW

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/excel/excel.chartaxis.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.chartaxis.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartseries.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.chartseries.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.charttitle.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.charttitle.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.customxmlpart.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.customxmlpart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.customxmlpartcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.customxmlpartcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.customxmlpartscopedcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.customxmlpartscopedcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.nameditem.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.nameditem.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.nameditemcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.nameditemcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.settingcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.settingcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheet.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.worksheet.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheetaddedeventargs.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.worksheetaddedeventargs.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheetcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.worksheetcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.ui.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.ui.yml) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-office/office.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/generate-docs/api-extractor-inputs-office/office.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/json/office.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/generate-docs/json/office.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/office.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/generate-docs/script-inputs/office.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/script-lab-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/generate-docs/script-inputs/script-lab-snippets.yaml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C14%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201811142057431089-242%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=kO6Ufa3%2F%2FjlcIMZ%2F9WvRgoMRr5JtJgw%2BEqdYMUOvoug%3D&st=2018-11-14T20%3A59%3A09Z&se=2018-12-15T21%3A04%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/docs-ref-autogen/excel/excel.charttitle.yml [hunk 1]
```diff
@@ -299,6 +312,19 @@ items:
       return:
         type:
           - number
+        description: |-
+
+          #### Examples
+
+          ```typescript
+          await Excel.run(async context => {
```
Comment (kbrandl) [inline docs/docs-ref-autogen/excel/excel.charttitle.yml]: `context` should have parens around it -> `await Excel.run(async (context) => {`

File: docs/docs-ref-autogen/excel/excel.customxmlpart.yml [hunk 2]
```diff
@@ -43,10 +43,10 @@ items:
           ```typescript
           await Excel.run(async (context) => {
               const settings = context.workbook.settings;
-              const xmlPartIDSetting = settings.getItem("ContosoReviewXmlPartId").load("value");
-
+              const xmlPartIDSetting = settings.getItemOrNullObject("ContosoReviewXmlPartId").load("value");
               await context.sync();
-              
+
+              if (xmlPartIDSetting.value) {
```
Comment (kbrandl) [inline docs/docs-ref-autogen/excel/excel.customxmlpart.yml]: Content within this `if` statement needs to be indented.

File: docs/docs-ref-autogen/excel/excel.customxmlpart.yml [hunk 3]
```diff
@@ -221,23 +220,22 @@ items:
           ```typescript
           await Excel.run(async (context) => {
               const settings = context.workbook.settings;
-              const xmlPartIDSetting = settings.getItem("ContosoReviewXmlPartId").load("value");
-
+              const xmlPartIDSetting = settings.getItemOrNullObject("ContosoReviewXmlPartId").load("value");
               await context.sync();
-              
+
+          if (xmlPartIDSetting.value) {
```
Comment (kbrandl) [inline docs/docs-ref-autogen/excel/excel.customxmlpart.yml]: This entire `if` block needs to be indented (within the `await Excel.run...` block).

File: docs/docs-ref-autogen/excel/excel.customxmlpartcollection.yml [hunk 4]
```diff
@@ -160,23 +159,22 @@ items:
           ```typescript
           await Excel.run(async (context) => {
               const settings = context.workbook.settings;
-              const xmlPartIDSetting = settings.getItem("ContosoReviewXmlPartId").load("value");
-
+              const xmlPartIDSetting = settings.getItemOrNullObject("ContosoReviewXmlPartId").load("value");
               await context.sync();
-              
+
+          if (xmlPartIDSetting.value) {
```
Comment (kbrandl) [inline docs/docs-ref-autogen/excel/excel.customxmlpartcollection.yml]: This entire `if` block needs to be indented (within the `await Excel.run...` block).

File: docs/docs-ref-autogen/excel/excel.customxmlpartscopedcollection.yml [hunk 5]
```diff
@@ -118,10 +118,10 @@ items:
           ```typescript
           await Excel.run(async (context) => {
               const settings = context.workbook.settings;
-              const xmlPartIDSetting = settings.getItem("ContosoReviewXmlPartId").load("value");
-
+              const xmlPartIDSetting = settings.getItemOrNullObject("ContosoReviewXmlPartId").load("value");
               await context.sync();
-              
+
+              if (xmlPartIDSetting.value) {
```
Comment (kbrandl) [inline docs/docs-ref-autogen/excel/excel.customxmlpartscopedcollection.yml]: Content within the `if` block needs to be indented.

File: docs/docs-ref-autogen/excel/excel.chartaxis.yml [hunk 6]
```diff
@@ -167,10 +167,12 @@ items:
 
               // Load to get display unit.
               valueAxis.load("displayUnit");
-             
+
               await context.sync();
 
-              OfficeHelpers.UI.notify("The vertical axis display unit is: " +  valueAxis.displayUnit);
+              OfficeHelpers.UI.notify(
+              "The vertical axis display unit is: " + valueAxis.displayUnit
```
Comment (kbrandl) [inline docs/docs-ref-autogen/excel/excel.chartaxis.yml]: this line needs to be indented

File: docs/docs-ref-autogen/excel/excel.range.yml [hunk 7]
```diff
@@ -1205,28 +1205,36 @@ items:
...
+          await Excel.run(async context => {
+              const sheet = context.workbook.worksheets.getItem("Sample");
               const salesTable = sheet.tables.getItem("SalesTable");
               const dataRange = salesTable.getDataBodyRange();
 
-              // Pass true so only cells with values count as used 
-              const usedDataRange = dataRange.getUsedRangeOrNullObject(true /* valuesOnly */);
+              // Pass true so only cells with values count as used
+              const usedDataRange = dataRange.getUsedRangeOrNullObject(
+              true /* valuesOnly */
```
Comment (kbrandl) [inline docs/docs-ref-autogen/excel/excel.range.yml]: this line needs to be indented

File: docs/docs-ref-autogen/excel/excel.range.yml [hunk 8]
```diff
@@ -1205,28 +1205,36 @@ items:
...
-                  OfficeHelpers.UI.notify("Need Data to Make Chart", "To create a meaningful chart, add names to the Product column and numbers to some of the other cells. Then press 'Try to create chart' again.");            
-              } else {            
-                  const chart = sheet.charts.add(Excel.ChartType.columnClustered, dataRange, "Columns");
-                  chart.setPosition("A15", "F30");
-                  chart.title.text = "Quarterly sales chart";
-                  chart.legend.position = "Right";
-                  chart.legend.format.fill.setSolidColor("white");
-                  chart.dataLabels.format.font.size = 15;
-                  chart.dataLabels.format.font.color = "black";
+              OfficeHelpers.UI.notify(
```
Comment (kbrandl) [inline docs/docs-ref-autogen/excel/excel.range.yml]: the `OfficeHelpers.UI.notify` lines need to be indented

File: docs/docs-ref-autogen/excel/excel.range.yml [hunk 9]
```diff
@@ -1205,28 +1205,36 @@ items:
...
+                  "Need Data to Make Chart",
+                  "To create a meaningful chart, press 'Fill the table' (or add names to the Product column and numbers to some of the other cells). Then press 'Try to create chart' again."
+              );
+              } else {
+              const chart = sheet.charts.add(
```
Comment (kbrandl) [inline docs/docs-ref-autogen/excel/excel.range.yml]: all lines within the `else` need to be indented

**Comment by AlexJerabek** [comment]
@kbrandl @ElizabethSamuel-MSFT 
I'll check in a change to office-js-snippets to fix these issues (and a couple others I noticed). I'll ping this thread and tag both of you for review when that happens.

**Comment by o365devx** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/docs-ref-autogen/excel/excel.chartaxis.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.chartaxis.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.chartseries.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.chartseries.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.charttitle.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.charttitle.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.conditionalformat.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.conditionalformat.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.conditionalformatcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.conditionalformatcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.customxmlpart.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.customxmlpart.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.customxmlpartcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.customxmlpartcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.customxmlpartscopedcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.customxmlpartscopedcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.nameditem.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.nameditem.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.nameditemcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.nameditemcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.range.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.range.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.settingcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.settingcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheet.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.worksheet.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheetaddedeventargs.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.worksheetaddedeventargs.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/excel/excel.worksheetcollection.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/excel/excel.worksheetcollection.yml) | :white_check_mark:Succeeded | |
[docs/docs-ref-autogen/office/office.ui.yml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/docs/docs-ref-autogen/office/office.ui.yml) | :white_check_mark:Succeeded | |
[generate-docs/api-extractor-inputs-office/office.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/generate-docs/api-extractor-inputs-office/office.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/json/office.api.json](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/generate-docs/json/office.api.json) | :white_check_mark:Succeeded | |
[generate-docs/json/snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/generate-docs/json/snippets.yaml) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/office.d.ts](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/generate-docs/script-inputs/office.d.ts) | :white_check_mark:Succeeded | |
[generate-docs/script-inputs/script-lab-snippets.yaml](https://github.com/OfficeDev/office-js-docs-reference/blob/elizs-get-latest/generate-docs/script-inputs/script-lab-snippets.yaml) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C11%5C15%5C0e1871f2-88cb-c16b-6d81-dc80e4f66cef%5CPullRequest%5C201811142357554516-242%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=SDJxsseagbCgUXC8GSGYpJA0dY8CuO%2FUpN3U5HRRvL0%3D&st=2018-11-14T23%3A59%3A25Z&se=2018-12-16T00%3A04%3A25Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/docs-ref-autogen/excel/excel.chartaxis.yml [hunk 10]
```diff
@@ -158,7 +158,7 @@ items:
           #### Examples
 
           ```typescript
-          await Excel.run(async (context) => {
+          await Excel.run(async context => {
```
Comment (ElizabethSamuel-MSFT) [inline docs/docs-ref-autogen/excel/excel.chartaxis.yml]: Alex is cleaning up these async context items. I'll merge after I get that update.

Comment (ElizabethSamuel-MSFT) [reply docs/docs-ref-autogen/excel/excel.chartaxis.yml]: Need to have another discussion as Script Lab is stripping out these non-whitespace characters. I'll go ahead and check this in in the meantime.

**Pull request merged**